samples/backend/csharp/Services/SemanticKernelApp.cs

```diff
@@ -66,10 +66,11 @@ internal SemanticKernelSession(Kernel kernel, IStateStore<string> stateStore, Gu
         ChatBot:";
 
     public async Task<AIChatCompletion> ProcessRequest(AIChatRequest message)
-    {        
+    {
         var chatFunction = _kernel.CreateFunctionFromPrompt(prompt);
```

Comment:
Better to cache this in a field rather than creating it every time.

Alternatively you don't even need a function here... you can just use string interpolation, e.g.
```C#
var botResponse = await _kernel.InvokePromptAsync($"""
    Prompt here that uses {history} and {userInput.Content}
    """);
```

Reply:
updated